resources/definitions/fdmprinter.def.json

```diff
@@ -4919,6 +4919,17 @@
                     "enabled": "magic_spiralize",
                     "settable_per_mesh": false,
                     "settable_per_extruder": false
+                },
+                "relative_extrusion":
+                {
+                    "label": "Use Relative Extrusion",
+                    "description": "Use relative extrusion rather than absolute extrusion. Before, Cura always used absolute extrusion. Most printers can cope with either relative or absolute extrusion.",
```

Comment:
How about:

Use relative extrusion rather than absolute extrusion. Using relative E-steps makes for easier post-processing of the Gcode. However, it's not supported by all printers and it may produce very slight differences in the printed object compared to using absolute E-steps.